masonry/src/doc/widget_properties.md

```diff
@@ -63,7 +63,7 @@ Some examples:
 - [`BorderColor`]
 - [`BorderWidth`]
 - [`Padding`]
-- [`TextColor`]
```

Comment:
We’re these previously in alphabetical ordering? Makes sense to reorder now.

Reply:
Nope, I just included a few as examples, it really doesn't matter.